lib/utils.js

```diff
@@ -101,3 +101,46 @@ exports.forceHttps = function(req, res, next) {
   }
   next();
 };
+
+// Middleware - store any data sent in session, and pass it to all views
+
+exports.autoStoreData = function (req, res, next) {
+
+  if (!req.session.data){
+    req.session.data = {};
+  }
+
+  for (var i in req.body){
+
+    // any input where the name starts with _ is ignored
+    if (i.indexOf("_") == 0){
```

Comment:
My linter complains that this should be `===`. I know it's not strictly needed, but worth doing so we don't get warnings?